tests/main.test.ts

```diff
@@ -141,6 +145,7 @@ function createContextInner(
 ): Context {
   return {
     eventName: "issue_comment.created",
+    //@ts-expect-error modified for testing purposes
```

Comment:
@gentlementlegen rfc

Reply:
You can also cast the object to the proper type, I believe it gives better auto-complete.